### PR TITLE
Warn users when URL is filtered out

### PR DESCRIPTION
**Pull request by hayesjordan**

### CODE REVIEW

File: android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java [hunk 1]
```diff
@@ -444,6 +448,36 @@ public void onError(String oldUrl) {
...
+          });
+        }
+
+        @Override
+        public void onPwsResult(PwsResult pwsResult) {
+        }
+
+        @Override
+        public void onPwsResultError(Collection<String> urls, int httpResponseCode, Exception e) {
+          if (urls.iterator().next().contains("https")){
```
Comment (cco3) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java]: startsWith

Comment (nondebug) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java]: We should toLower the URL before checking the scheme so we can match (eg) HTTPS://www.google.com

File: android/PhysicalWeb/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -64,5 +64,7 @@
     <string name="url_charset_error">Non-ASCII characters unsupported in URL. Please use a shortener.</string>
     <string name="error_bluetooth_support">Device does not support Bluetooth</string>
     <string name="error_api_key_no_longer_available">Google API key no longer available, please update settings.</string>
+    <string name="pws_filtered_url_unknown">Other clients may filter your URL.</string>
+    <string name="pws_filtered_url_http">Other clients may filter your URL because it is not https.</string>
```
Comment (nondebug) [inline android/PhysicalWeb/app/src/main/res/values/strings.xml]: Let's capitalize HTTPS in the warning string

**Comment by nondebug** [comment]
lgtm

**Pull request merged**